pubspec.yaml

```diff
@@ -25,4 +25,6 @@ dev_dependencies:
 
 
 flutter:
-  uses-material-design: true
\ No newline at end of file
+  uses-material-design: true
+  assets:
+    - assets/
```

Comment:
put in presentation layer

Reply:
disagree

Reply:
u use assets only in presentation module, agree with Egor

Reply:
I asked @alexei-kruk-idf  and he said put it here, not in presentation.

Reply:
it's ok. u can keep it in root's pubspec